{"name": "Логин с неверным логином/паролем (параметризация)", "status": "passed", "steps": [{"name": "Авторизируемся в приложении", "status": "passed", "start": 1750094140683, "stop": 1750094140998}], "parameters": [{"name": "wrong_logout_data", "value": "{'email': 'ktrof_006@yandex.ru', 'password': 'wrong_ktrof_006@yandex.ru'}"}], "start": 1750094140683, "stop": 1750094140998, "uuid": "0ca63f76-8f2e-43e8-b9bb-1a2038b34e93", "historyId": "1a8158e5dbbb18c24565e67ed54ea5aa", "testCaseId": "04493e2aa7ea31843048e03f3ff14117", "fullName": "tests.test_login.TestLogin#test_login_with_wrong_data_status_code_401", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Rumata"}, {"name": "thread", "value": "22736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}